### DISCUSSION TITLE
2005/06/02/%E4%B8%80%E5%A4%A7%E5%8F%91%E7%8E%B0

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#34] [Announcements]
# 2005/06/02/%E4%B8%80%E5%A4%A7%E5%8F%91%E7%8E%B0

我在标题里想写“查阅和退定联通sp服务”，结果 MSN space 就是不让我发表，我只好在联通两个字之间加了一个小横线。

https://blog.qizhen.xyz/2005/06/02/%E4%B8%80%E5%A4%A7%E5%8F%91%E7%8E%B0

**Comment by ruanqizhen** [comment]
婵
2005年06月8日 13:56 编辑
今天来看看你有没有偷懒没作功课！嘻嘻！不错不错！联通和微软应该是程序问题吧…………不晓得！猜测…………

回复
qingxi
2005年06月7日 18:40 编辑
我也“援助”两字都不能发表，你讲奇怪不？

回复
静
2005年06月6日 08:35 编辑
微软和联通有虾米仇呢～？？？

回复
颠簸中奔向希望
2005年06月4日 21:06 编辑
flash今天好象不能贴了，语句被屏蔽了

回复
syren
2005年06月3日 14:57 编辑
呵呵~这么搞笑啊~！·

回复
Radon
2005年06月3日 12:15 编辑
真的是很不错的space哦！而且也很大方，把知道的都告诉我们，学到很多东西耶……先在此谢过咯…

回复
shielf
2005年06月3日 11:54 编辑
记事本那个bug是挺搞笑的

回复
权育
2005年06月2日 21:18 编辑
有这样一回事?我还是头一次知道.

回复
旦阳
2005年06月2日 20:30 编辑
真的啊？就像windows记事本输入“联通”变成块烧焦的电板？

回复
